Eastern-Research-Group/mywaterway
# Feature/updated protect tab

## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3610789

## Main Changes:
* Adds the new Protect Tab content including the WSIO, PAD, and Wild and Scenic Rivers data.
* The Protected Areas Database service currently returns a CORS error when developing locally so it is not fully implemented. Once the service is fixed we can finish implementing the feature. For now the app attempts to query the service and handles the CORS error correctly.

## Steps To Test:
1. Navigate to http://localhost:3000/community/troutdale/protect
2. Check that the WSIO and Wild and Scenic River accordions are working correctly.
3. Check that the Protected Areas accordion correctly displays an error message.
